src/xiyouji.html

```diff
@@ -0,0 +1,184 @@
+<!doctype html>
```

Comment:
I think the name of this fiile (xiyouji.html) needs to start with "example" to meet the Makefile rule?

Reply:
Turns out no; the Make rule is:

```
www/examples/%.html: src/%.html
	cp $< www/examples
```

It works:

```
~/wbe $ make www/examples/xiyouji.html
cp src/xiyouji.html www/examples
```